libraries/sbg-rs/src/sbg.rs

```diff
@@ -258,7 +258,7 @@ impl SBG {
     /**
      * Allows the SBG interface to read data from the serial ports.
      */
-    pub extern "C" fn SbgInterfaceReadFunc(
+    pub unsafe extern "C" fn SbgInterfaceReadFunc(
```

Comment:
Why encase the function with a whole unsafe block?

Reply:
This Clippy error was getting triggered: https://rust-lang.github.io/rust-clippy/master/index.html#not_unsafe_ptr_arg_deref

Reply:
Sounds good.